modules/nf-core/coptr/estimate/main.nf

```diff
@@ -0,0 +1,49 @@
+process COPTR_ESTIMATE {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/coptr:1.1.4--pyhdfd78af_3':
+        'biocontainers/coptr:1.1.4--pyhdfd78af_3' }"
+
+    input:
+    tuple val(meta), path(pkl, stageAs: "coverage_maps/*")
```

Comment:
Does this have to be a directory?

Reply:
It does, this tool is a bit particular that way...

Reply:
OK